android/app/src/main/java/app/organicmaps/bookmarks/data/BookmarkCategory.java

```diff
@@ -105,17 +105,15 @@ public String getDescription()
   @Override
   public String toString()
   {
-    final StringBuilder sb = new StringBuilder("BookmarkCategory{");
-    sb.append("mId=").append(mId);
```

Comment:
Are you sure that the nee code is faster? Is there any documentation or benchmark about it?

Reply:
You're right
In the most of case, StringBuilder is more efficient -> https://dip-mazumder.medium.com/stringbuilder-vs-string-in-java-a-guide-for-optimal-memory-usage-4a284d8243ea